tests/cross_dock/test_integration.py

```
@@ -0,0 +1,70 @@
+Integration tests for cross-dock functionality.
+
+This module contains integration tests for the cross-dock functionality,
+testing the full workflow from file input to Excel output.
+"""
+
+import os
+import tempfile
+from unittest import mock
+
+import pandas as pd
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)